script/_dns64

```diff
@@ -82,7 +82,9 @@ dns64_install()
 
     service_name="$(_detect_service_name)"
 
-    [ -f /.dockerenv ] || sudo sh -c "echo \"nameserver $DNS64_NAMESERVER_ADDR\" >> $RESOLV_CONF_HEAD"
+    if without DOCKER; then
```

Comment:
I tried all of the approaches below and they did not work in this context for some reason:

- `-f /.dockerenv`
- `grep docker /proc/1/cgroup -qa`
- `grep 'docker\|lxc' /proc/1/cgroup -qa`